bin/mailing-common/mailing-list-utils.rb

```diff
@@ -176,3 +176,18 @@ def generate(name)
 
   deduped_csv
 end
+
+
+def query_all_emails_at_domain(domain)
+  puts "Emails at #{domain}"
+
+  {}.tap do |results|
+    DB[:contacts].where(Sequel.ilike(:email,"%@#{domain}")).distinct.select(:name, :email).each do |contact|
```

Comment:
good point. I presume that email & name is 1:1, but it may not be the case. Anyway like you said they're all deduped by email later so it's ok